### PR TITLE
[pretty-time] Add typing for pretty-time

### PR DESCRIPTION
**Pull request by azerella**
@jonschlinkert Adding some type defs for `pretty-time` ❤️ 

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/pretty-time/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    | 'hours'
+    | 'd'
+    | 'day'
+    | 'days'
+    | 'w'
+    | 'wk'
+    | 'week'
+    | 'weeks';
+
+export default function(time: number[], smallest?: Increment, digits?: number): string;
```
Comment (plantain-00) [inline types/pretty-time/index.d.ts]: It should be `export =`: https://github.com/jonschlinkert/pretty-time/blob/master/index.js#L12

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/pretty-time/index.d.ts (hunk 1 above)
Comment (adamzerella) [reply types/pretty-time/index.d.ts]: @plantain-00 I'm not sure how to export a default function with `export = `. Should I be declaring a `module` ?

Comment (plantain-00) [reply types/pretty-time/index.d.ts]: See:
http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html
https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/pretty-time/index.d.ts (hunk 1 above)
Comment (adamzerella) [reply types/pretty-time/index.d.ts]: Ahh gotcha! updated 😃

**Review by plantain-00 — APPROVED** [review]

**Pull request merged**